api/graphql/schema.graphql

```diff
@@ -143,6 +143,8 @@ type User {
 
   # Bookmarks
   poolBookmarks: [Pool] @belongsToMany
+  # My talent requests
+  poolCandidateSearchRequests: [PoolCandidateSearchRequest!]
```

Comment:
No guards on this.  Assuming if you can see the user you can see their requests as well.

Reply:
I don't think this is a safe assumption, actually. We allow managers to sometimes view a candidate's "user profile". Anything extraneous to their profile data (like requests they've submitted) should be protected separately.

Reply:
Good idea.  I've add a canResolved on the view policy.
[add guard to user property](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11653/commits/a6d3e32b0ac5153869fa9d20fbebc005bfdf0136)